README.md

```diff
@@ -5,18 +5,18 @@ Federation is fully supported.
 
 Based on preliminary testing, it is possible to achieve a significant
 throughput improvement at the expense of the freshness of the data.
-Setting the ttl accordingly is of critical importance.
+Setting the ttl accordingly and/or a good invalidation strategy is of critical importance.
 
-Under the covers it uses [`async-cache-dedupe`](https://github.com/mcollina/async-cache-dedupe)
+Under the covers, it uses [`async-cache-dedupe`](https://github.com/mcollina/async-cache-dedupe)
 which will also deduplicate the calls.
 
 ## Install
 
 ```bash
-npm i fastify mercurius mercurius-cache
+npm i fastify mercurius mercurius-cache graphql
```

Comment:
why this change?

Reply:
because of this https://github.com/mercurius-js/mercurius/issues/693#issuecomment-1000863371